src/core/client/admin/components/ModerateCard/ModerateCardDetailsContainer.tsx

```diff
@@ -0,0 +1,90 @@
+import { Localized } from "fluent-react/compat";
+import React, { FunctionComponent, useState } from "react";
+import { graphql } from "react-relay";
+
+import { ModerateCardDetailsContainer_comment as CommentData } from "coral-admin/__generated__/ModerateCardDetailsContainer_comment.graphql";
+import { ModerateCardDetailsContainer_settings as SettingsData } from "coral-admin/__generated__/ModerateCardDetailsContainer_settings.graphql";
+import { withFragmentContainer } from "coral-framework/lib/relay";
+import { Flex, HorizontalGutter, Icon, Tab, TabBar } from "coral-ui/components";
+
+import CommentRevisionContainer from "./CommentRevisionContainer";
+import FlagDetailsContainer from "./FlagDetailsContainer";
+import styles from "./ModerateCardDetailsContainer.css";
+
+interface Props {
+  comment: CommentData;
+  settings: SettingsData;
+  onUsernameClick: (id?: string) => void;
+  hasDetails: boolean;
+  hasRevisions: boolean;
+}
+
+const ModerateCardDetailsContainer: FunctionComponent<Props> = ({
+  comment,
+  onUsernameClick,
+  settings,
+  hasDetails,
+  hasRevisions,
+}) => {
+  const [activeTab, setActiveTab] = useState<"DETAILS" | "HISTORY">(
+    hasDetails ? "DETAILS" : "HISTORY"
+  );
+
+  return (
+    <HorizontalGutter>
+      <TabBar
+        variant="secondary"
+        activeTab={activeTab}
+        onTabClick={id => setActiveTab(id as "DETAILS" | "HISTORY")}
```

Comment:
If repeating `"DETAILS | "HISTORY"` becomes arduous, you can declare a `Tabs` type for it. :)

```
type Tabs = "DETAILS" | "HISTORY";
```